src/workerd/api/pyodide/pyodide.c++

```diff
@@ -317,40 +317,64 @@ kj::Array<kj::String> ArtifactBundler::parsePythonScriptImports(kj::Array<kj::St
   return result.releaseAsArray();
 }
 
-// This is equivalent to `pkgImport.replace('.', '/') + ".py"`.
-kj::String importToModuleFilename(kj::StringPtr pkgImport) {
-  auto result = kj::heapString(pkgImport.size() + 3);
-  for (auto i = 0; i < pkgImport.size(); i++) {
-    if (pkgImport[i] == '.') {
-      result[i] = '/';
-    } else {
-      result[i] = pkgImport[i];
-    }
-  }
+kj::Array<kj::StringPtr> snapshotImports = kj::arr("_pyodide"_kj,
+    "_pyodide.docstring"_kj,
+    "_pyodide._core_docs"_kj,
+    "traceback"_kj,
+    "collections.abc"_kj,
+    // Asyncio is the really slow one here. In native Python on my machine, `import asyncio` takes ~50
+    // ms.
+    "asyncio"_kj,
+    "inspect"_kj,
+    "tarfile"_kj,
+    "importlib",
+    "importlib.metadata"_kj,
+    "re"_kj,
+    "shutil"_kj,
+    "sysconfig"_kj,
+    "importlib.machinery"_kj,
+    "pathlib"_kj,
+    "site"_kj,
+    "tempfile"_kj,
+    "typing"_kj,
+    "zipfile"_kj);
 
-  result[pkgImport.size()] = '.';
-  result[pkgImport.size() + 1] = 'p';
-  result[pkgImport.size() + 2] = 'y';
-  return result;
+kj::Array<kj::StringPtr> ArtifactBundler::getSnapshotImports() {
+  return kj::heapArray((kj::ArrayPtr<kj::StringPtr>)snapshotImports);
```

Comment:
@danlapid this seems to be another place where it can't figure out what to do: `kj::heapArray()` accepts an `ArrayPtr` and I'm passing an `Array` but it doesn't seem to automatically decay to `ArrayPtr`.

Reply:
Just return an ArrayPtr instead

Reply:
Well this is passing out to JavaScript so I think if I combined making it `const` with this change, it should hopefully be a compile error? Because there isn't a way to prove that it is thread safe?

Reply:
Either that or jsg will do the copy in the backend? 😁

Reply:
Not only does it not work, it doesn't seem possible to cast `const kj::Array<...>` to `const kj::ArrayPtr<...>` so instead I have to do:
```C++
kj::heapArray(snapshotImports.begin(), snapshotImports.size())
```